src/tests/Common/dirs.proj

```diff
@@ -71,6 +71,10 @@
       <_GroupStartsWith Include="$(TestRoot)JIT\Regression\CLR-x86-JIT\V1-M12-Beta2\b35351\b35351.ilproj">
         <GroupNumber>3</GroupNumber>
       </_GroupStartsWith>
+
+      <_GroupStartsWith Include="$(TestRoot)tracing\eventpipe\">
+        <GroupNumber>4</GroupNumber>
+      </_GroupStartsWith>
```

Comment:
Why are we adding a new test build group here? Was this just for debugging?

Reply:
This PR allows running only a specific subset of runtime tests on iOS platforms, identified by group number 4. The default testing group, which includes a large set of runtime tests results in a timeout when attempting to build and AOT compile in the `BuildManagedTestGroup` target.

Is there an alternative approach to building and AOT compiling only a subset of tests?

This PR sets up infrastructure to run an iOS runtime test on the CI. As a next step, we plan to introduce parallel AOT compilation to allow all runtime tests to run, and remove filtering through the testing groups.

Reply:
Yes, there's the `tree` argument to build a subteee of tests, where you can specify a subteee of tests to build.

The group concept here doesn't actually do subsets. It basically makes slices in alphabetical ordering that start at the provided test and end at the start of the next group.

Reply:
You can limit the subtree in the build script instead of in the actual MSBuild code (this is what NativeAOT does). I'll share an example when I'm at a computer.

Reply:
Here's an example from runtime.yml on how to pass the arguments in:

https://github.com/dotnet/runtime/blob/b9df3643c41b89dd211eeef54eb69799ad3b6e4f/eng/pipelines/runtime.yml#L211

Reply:
Thanks! I found the `__BuildTestTree`, so as you suggested we will enable the subset of tests by using the subtree.